.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: java
+jdk: oraclejdk8
+
+script:
+  - mvn test --batch-mode
+  - mvn checkstyle:check --batch-mode --fail-never
```

Comment:
> What's `--fail-never` doing?

`--fail-never` prevents the build from failing when a violation happens.

> Shouldn't it failOnViolation so that the checks on GitHub will turn red when a PR regresses the Google Style adherence?

It should.
@drewwills requested that checkstyle not be a breaking part of the build until uPortal/portlets can pass the lint checks.